### PR TITLE
Release `v6.0.2`

### LINKED ISSUE #1072: Remove deprecated storage
https://github.com/darwinia-network/darwinia/blob/94a0a2a4e46ebf01fff5f352f14e30517dd3648b/pallet/ecdsa-authority/src/lib.rs#L174-L184

### PR DESCRIPTION
**Pull request by AurevoirXavier**
The goals of this release are:
- test `polkadot-v0.9.38`
- add multisig migration event for front-end

Closes #1072.

### CODE REVIEW

File: pallet/ecdsa-authority/src/lib.rs [hunk 1]
```diff
@@ -224,15 +212,6 @@ pub mod pallet {
 				Default::default()
 			}
 		}
-
-		fn on_runtime_upgrade() -> Weight {
```
Comment (AurevoirXavier) [inline pallet/ecdsa-authority/src/lib.rs:228]: It's safe to remove for Crab.

Because we haven't setup any ecdsa authority in Crab yet.

No data on Crab.

**Review by boundless-forest — DISMISSED** [review]
Please fix the CI. The changes LGTM

File: pallet/ecdsa-authority/src/lib.rs (hunk 1 above)
Comment (boundless-forest) [reply pallet/ecdsa-authority/src/lib.rs:228]: The current Pangoro on chain runtime version is `6,009`. Does it need the ecdsa migration? Maybe you should upgrade that first.

**Comment by AurevoirXavier** [comment]
We have the existing release files for Pangoro. It's okay to remove it in this version.

**Comment by AurevoirXavier** [comment]
Draft.

I think there are some account migration issues now.

**Review by boundless-forest — APPROVED** [review]

**Pull request merged**